test/git_test.py

```diff
@@ -0,0 +1,35 @@
+import logging
+import shutil
+import unittest
+
+from info_processing.git.helpers import get_repository_info
```

Comment:
wrong order of imports. It should be:
* standard library
* external dependencies
* from this module

Reply:
linter has another opinion about that